charts/postgres-operator/crds/operatorconfigurations.yaml

```diff
@@ -64,6 +64,8 @@ spec:
               type: boolean
             enable_shm_volume:
               type: boolean
+            should_delete_unused_pvc:
```

Comment:
would keep the pattern of naming flags `enable...`, maybe `enable_unused_pvc_deletion` or something (plus, mind alphabetical order)